langs/go.go

```diff
@@ -18,8 +18,8 @@ func (h *GoLangHelper) Runtime() string {
 	return h.LangStrings()[0]
 }
 
-func (h *GoLangHelper) DefaultFormat() string {
-	return "http-stream"
+func (h *GoLangHelper) CustomMemory() uint64 {
```

Comment:
not sure why most of the lang helpers don't embed a `BaseHelper` in their struct? stuff like this can just inherit and doesn't need to modify for most langs. kind of a nit I guess, most lang's methods seem to return different stuff